src/helm/benchmark/annotation/annotator.py

```diff
@@ -20,6 +20,11 @@ def annotate(self, request_state: RequestState) -> Any:
         that are implementation specific."""
         pass
 
+    def annotate_all(self, request_states: List[RequestState]) -> Any:
```

Comment:
If the function is a mutator, the return type should be `-> None`. Or you can make this return a list of annotations (specify the actual type rather than `Any`).

Reply:
Currently making this return a list of annotations and modified the typing to `-> List[Dict[str, Any]]`, are there any specific annotation types that should be used? or should I create a new class for annotations?